src/screenComponents/radarView.cpp

```diff
@@ -600,27 +608,34 @@ void GuiRadarView::drawObjects(sf::RenderTarget& window_normal, sf::RenderTarget
     case FriendlysShortRangeFogOfWar:
         if (!my_spaceship)
             return;
+
         foreach(SpaceObject, obj, space_object_list)
         {
             if (!obj->canHideInNebula())
                 visible_objects.insert(*obj);
 
-            if ((!P<SpaceShip>(obj) && !P<SpaceStation>(obj))
+            P<ShipTemplateBasedObject> stb_obj = obj;
+
+            if (!stb_obj
                 || (!obj->isFriendly(my_spaceship) && obj != my_spaceship))
             {
                 P<ScanProbe> sp = obj;
+
                 if (!sp || sp->owner_id != my_spaceship->getMultiplayerId())
                 {
                     continue;
                 }
             }
 
+            float r = (stb_obj && obj->isFriendly(my_spaceship)) ? stb_obj->getShortRangeRadarRange() : 5000.0f;
```

Comment:
Not sure why the `obj->isFriendly(my_spaceship)` check is in here?

Reply:
It's redundant; removed it